spec/models/concerns/devise_overrides_spec.rb

```diff
@@ -0,0 +1,174 @@
+require 'rails_helper'
+
+describe DeviseOverrides do
+  let(:extended_class) { Class.new { extend ActiveSupport::Concern } }
+  
+  describe '.confirmed?' do
+    let(:customer) { Customer.new(password: '123451', password_confirmation: '123451', email: 'test@gmail.com') }
+    it 'return confirmed status' do
+      expect(customer.confirmed?).to eq(customer.active?)
+    end
+  end
+  
+  describe '.confirmation_required?' do
```

Comment:
This method is being used by devise internally to generate confirmation_token and sending confirmation instruction to the user during signup process. But the confirmation instruction is being sent by the API itself, so this method should always return false in our case.